recognition/include/pcl/recognition/face_detection/rf_face_detector_trainer.h

```diff
@@ -45,35 +45,18 @@ namespace pcl
       using NodeType = face_detection::RFTreeNode<face_detection::FeatureType>;
       pcl::DecisionForest<NodeType> forest_;
 
-      std::string model_path_;
-      bool pose_refinement_;
+      std::string model_path_ {"face_mesh.ply"};
+      bool pose_refinement_ {false};
       int icp_iterations_;
 
       pcl::PointCloud<pcl::PointXYZ>::Ptr model_original_;
-      float res_;
+      float res_ {0.005f};
 
     public:
 
       RFFaceDetectorTrainer()
       {
-        w_size_ = 80;
-        max_patch_size_ = 40;
-        stride_sw_ = 4;
-        ntrees_ = 10;
-        forest_filename_ = std::string ("forest.txt");
-        nfeatures_ = 10000;
-        thres_face_ = 1.f;
-        num_images_ = 1000;
-        trans_max_variance_ = 1600.f;
-        used_for_pose_ = std::numeric_limits<int>::max ();
-        use_normals_ = false;
-        directory_ = std::string ("");
-        HEAD_ST_DIAMETER_ = 0.2364f;
-        larger_radius_ratio_ = 1.5f;
         face_heat_map_.reset ();
```

Comment:
Is there a way to replace face_heat_map_.reset ();? Here  https://cplusplus.com/reference/memory/shared_ptr/reset/ it says For signature (1) the object becomes empty (as if [default-constructed](https://cplusplus.com/shared_ptr::shared_ptr)) so either { nullptr } or {} should be fine

Reply:
Yes, let's use `{}`